InfiniteRecharge/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+
+import frc.robot.RobotMap;
+
+public class Intake extends Subsystem {
+
+
+
+    public double getPosition() {
+        return 0.0; //position
+      
+    }
+
+    public boolean zeroPosition() {
+        return false;
+
+    }
+
+    public boolean extendIntake() {
+        //Motion profile stuff
+        RobotMap.getIntakeActuationSystem().set(1);
+        return false;
+    }
+    public boolean retractIntake() {
+        //Motion profile stuff
+        RobotMap.getIntakeActuationSystem().set(-1);
```

Comment:
i think running the intake extension at full power all the time would result in some "interesting" robot behaviour. we can look into motion profiling later soon though :p

Reply:
a possible solution could be to use the WPILib's new ProfiledPIDController (or even just a proportional control loop)